src/azure-cli/azure/cli/command_modules/vm/_vmss_application_health.py

```diff
@@ -0,0 +1,60 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# vmss application health setting for rolling upgrade policy
+
+application_health_setting_for_linux = '''
+#cloud-config
+package_upgrade: true
+
+packages:
+  - nginx
+
+write_files:
+  - path: /var/www/html/index.html
+    content: |
+      <!DOCTYPE html>
+      <html>
+      <body>
+      <h1>Hello, World!</h1>
+      </body>
+      </html>
+    owner: www-data:www-data
```

Comment:
Just out of curiosity, I would like to ask why we need this file?

Reply:
Rolling upgrade mode requires setting it up to use.